vllm/entrypoints/openai/rpc/__init__.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from typing import Optional, Mapping
+from enum import Enum
+
+from vllm.inputs import PromptInputs
+from vllm.lora.request import LoRARequest
+from vllm.prompt_adapter.request import PromptAdapterRequest
+from vllm.sampling_params import SamplingParams
+
+VLLM_GENERATE_RPC_PATH = "tcp://localhost:5570"
+VLLM_GET_DATA_RPC_PATH = "tcp://localhost:5571"
+VLLM_IS_READY_RPC_PATH = "tcp://localhost:5572"
```

Comment:
At the very least we should be choosing randomly available ports above the "user space" of the 1000s port range. See https://pyzmq.readthedocs.io/en/latest/api/zmq.html#zmq.Socket.bind_to_random_port

Reply:
@mgoin yeah that'd be better. Any idea how we'd notify the clients what port to connect to in that case?

The server that would be calling `.bind_to_random_port()` is in a different process than the openai server that needs to connect clients to it

Reply:
the the openai server can listen first, pass the ports to the server process, and then the server just connects to it?